README.md

```diff
@@ -44,6 +44,10 @@ use DeviceDetector\Parser\Device\AbstractDeviceParser;
 // for other options see VERSION_TRUNCATION_* constants in DeviceParserAbstract class
 AbstractDeviceParser::setVersionTruncation(AbstractDeviceParser::VERSION_TRUNCATION_NONE);
 
+// needed for clientHints
+header('Accept-CH: Sec-CH-UA-Full-Version, Sec-CH-UA-Platform, Sec-CH-UA-Platform-Version, Sec-CH-UA-Model, Sec-CH-UA-Arc', true);
+
+
```

Comment:
We maybe should include a new section about client hints overall instead, also explaining how they work, and that they won't fully work on the first request,.. So maybe remove it again here and we'll create another PR to add some more docs on that later.